src/webapi/document.rs

```diff
@@ -21,6 +21,17 @@ use private::TODO;
 #[reference(subclass_of(EventTarget, Node))]
 pub struct Document( Reference );
 
+error_enum_boilerplate! {
+    CreateElementError,
+    InvalidCharacterError
+}
```

Comment:
So I guess these can't actually throw `NotSupportedError`? (Reading the specs more closely it seems like it might be the case.) If so then we don't need this enum - we can just return `InvalidCharacterError` from `create_element` directly. (In general those enums are when we have 2 or more possible exceptions.)

Reply:
Updated!